costs/costs_route.go

```diff
@@ -178,7 +178,7 @@ func parseFormAndQueryParams(request *http.Request) (esQueryParams, error) {
 	return parsedParams, nil
 }
 
-func makeElasticSearchRequestAndParseIt(ctx context.Context, parsedParams esQueryParams, user users.User) (es.SimplifiedCostsDocument, error) {
+func makeElasticSearchRequestAndParseIt(ctx context.Context, parsedParams esQueryParams, user users.User) (es.SimplifiedCostsDocument, int, error) {
```

Comment:
Because unlike `error` and `es.SimplifiedCostDocument`, `int` does not inform on the purpose of what it contains, perhaps you should document what this function returns.